DecipherNow/gm-fabric-dashboard
# feat: generate GitHub post body

## Proposed Changes

  - Part One of automation to generate the GitHub Post body. Follow on work is dependent on figuring out our approach for management secrets

## How to Test

  -Run `npm run generateGitHubDraft` and confirm that the POST body makes sense and has the right content
  - Click through to Regexr and understand how my crazy expression works
